packages/react-reconciler/src/ReactCurrentFiber.js

```diff
@@ -91,6 +91,7 @@ export function resetCurrentFiber() {
 export function setCurrentFiber(fiber: Fiber) {
   if (__DEV__) {
     ReactDebugCurrentFrame.getCurrentStack = getCurrentFiberStackInDev;
+    ReactDebugCurrentFrame.currentFiber = fiber;
```

Comment:
We don't want to expose the fiber itself. This module is meant to work with different implementations too. For example server renderer sets its own `ReactDebugCurrentFrame.getCurrentStack` implementation that doesn't use Fibers internally.

I was thinking something more like 

```js
ReactDebugCurrentFrame.getCurrentStackFrames = ...
```

which returns the object. Set it together with `ReactDebugCurrentFrame.getCurrentStack`. Later we'll remove `ReactDebugCurrentFrame.getCurrentStack`.

Reply:
ok. i guess i shoudl have asked about this haha, i was also unsure about it. i think i understand.